<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>LiuBao</author>
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PMS</string>
  </property>
  <widget class="QTabWidget" name="centralTabwidget">
   <property name="tabsClosable">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>视图(&amp;V)</string>
    </property>
    <addaction name="toolBarAct"/>
    <addaction name="classTreeAct"/>
    <addaction name="statisticsTreeAct"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="aboutAct"/>
   </widget>
   <widget class="QMenu" name="configMenu">
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
    <addaction name="classMgmtAct"/>
    <addaction name="configAct"/>
   </widget>
   <widget class="QMenu" name="dataMenu">
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>数据(&amp;D)</string>
    </property>
    <widget class="QMenu" name="exportMenu">
     <property name="title">
      <string>导出到</string>
     </property>
     <property name="icon">
      <iconset resource="../../Resource.qrc">
       <normaloff>:/Icon/image/table-export.png</normaloff>:/Icon/image/table-export.png</iconset>
     </property>
     <addaction name="toPDFAct"/>
     <addaction name="toExcelAct"/>
    </widget>
    <addaction name="printAct"/>
    <addaction name="exportMenu"/>
    <addaction name="exitAct"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="cutAct"/>
    <addaction name="copyAct"/>
    <addaction name="pasteAct"/>
    <addaction name="delAct"/>
   </widget>
   <addaction name="dataMenu"/>
   <addaction name="editMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="configMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>宋体</family>
    </font>
   </property>
  </widget>
  <widget class="QDockWidget" name="navigationDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>导航栏</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QTreeWidget" name="navigationTree">
       <property name="font">
        <font>
         <family>宋体</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QTreeView::item{height:25px}</string>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <property name="indentation">
        <number>25</number>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="statisticsDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>统计信息</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QTreeView" name="statisticsTree">
       <property name="font">
        <font>
         <family>宋体</family>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="aboutAct">
   <property name="icon">
    <iconset resource="../../Resource.qrc">
     <normaloff>:/Icon/image/information.png</normaloff>:/Icon/image/information.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="printAct">
   <property name="icon">
    <iconset resource="../../Resource.qrc">
     <normaloff>:/Icon/image/printer.png</normaloff>:/Icon/image/printer.png</iconset>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="toPDFAct">
   <property name="icon">
    <iconset resource="../../Resource.qrc">
     <normaloff>:/Icon/image/page_white_acrobat.png</normaloff>:/Icon/image/page_white_acrobat.png</iconset>
   </property>
   <property name="text">
    <string>PDF文件</string>
   </property>
  </action>
  <action name="toExcelAct">
   <property name="icon">
    <iconset resource="../../Resource.qrc">
     <normaloff>:/Icon/image/page_excel.png</normaloff>:/Icon/image/page_excel.png</iconset>
   </property>
   <property name="text">
    <string>Excel文件</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="icon">
    <iconset resource="../../Resource.qrc">
     <normaloff>:/Icon/image/exit.png</normaloff>:/Icon/image/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="toolBarAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>工具栏</string>
   </property>
  </action>
  <action name="classTreeAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>导航栏</string>
   </property>
  </action>
  <action name="statisticsTreeAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>统计信息</string>
   </property>
  </action>
  <action name="copyAct">
   <property name="icon">
    <iconset resource="../../Resource.qrc">
     <normaloff>:/Icon/image/copy.png</normaloff>:/Icon/image/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="pasteAct">
   <property name="icon">
    <iconset resource="../../Resource.qrc">
     <normaloff>:/Icon/image/paste.png</normaloff>:/Icon/image/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="cutAct">
   <property name="icon">
    <iconset resource="../../Resource.qrc">
     <normaloff>:/Icon/image/cut.png</normaloff>:/Icon/image/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="delAct">
   <property name="icon">
    <iconset resource="../../Resource.qrc">
     <normaloff>:/Icon/image/delete.png</normaloff>:/Icon/image/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="configAct">
   <property name="icon">
    <iconset resource="../../Resource.qrc">
     <normaloff>:/Icon/image/gear.png</normaloff>:/Icon/image/gear.png</iconset>
   </property>
   <property name="text">
    <string>首选项</string>
   </property>
  </action>
  <action name="classMgmtAct">
   <property name="text">
    <string>班级管理</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
